sshd_config

```diff
@@ -1,6 +1,9 @@
 PasswordAuthentication no
 PermitEmptyPasswords no
 
+PubkeyAcceptedKeyTypes +ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-rsa,ssh-dss
+HostkeyAlgorithms +ssh-rsa
```

Comment:
Prepending the list with a plus will **add** those. Omitting the plus would **replace** the list.